cluster/juju/layers/kubernetes-master/templates/addons/dashboard-controller.yaml

```diff
@@ -0,0 +1 @@
+# Empty placeholder template, to be replaced at build time
```

Comment:
The placeholder templates are the bit I don't like, and I couldn't find a way around it.

The problem is, if these files don't exist when `charm build` is run, then it won't pick 'em up. This leaves us with two options:

1. Have the `AddonsTactic` copy the addon files in manually
2. Use placeholder templates and let charm build copy them

Option 1 doesn't work because `charm build` will fuss about it on subsequent builds:
```
build: Composing into /home/gkk/work/juju-repo/charms
build: Destination charm directory: /home/gkk/work/juju-repo/charms/builds/kubernetes-master
build: Added unexpected file, should be in a base layer: templates/addons/dashboard-service.yaml
build: Added unexpected file, should be in a base layer: templates/addons/heapster-controller.yaml
build: Added unexpected file, should be in a base layer: templates/addons/influxdb-service.yaml
build: Added unexpected file, should be in a base layer: templates/addons/kubedns-rc.yaml
build: Added unexpected file, should be in a base layer: templates/addons/dashboard-controller.yaml
build: Added unexpected file, should be in a base layer: templates/addons/heapster-service.yaml
build: Added unexpected file, should be in a base layer: templates/addons/grafana-service.yaml
build: Added unexpected file, should be in a base layer: templates/addons/influxdb-grafana-controller.yaml
build: Added unexpected file, should be in a base layer: templates/addons/kubedns-svc.yaml
build: Unable to continue due to unexpected modifications (try --force)
```

Which leaves us with option 2, placeholders. I don't like it because we're gonna be tripping over that with git commits all the time, and our list of addon manifests is now maintained in 3 different places. Whee.